docs/guides/admin/docs/modules/studio.md

```diff
@@ -14,6 +14,23 @@ The path `/studio` is accessible by users with the role `ROLE_ADMIN` or `ROLE_ST
 
 The preferred way to let your users access Studio is via LTI. Remember to configure your LTI users to have the role `ROLE_STUDIO` so that they can access Studio and all APIs used by Studio.
 
+This can be done, for example, by configuring the `LtiLaunchAuthenticationHandler` in `etc/org.opencastproject.security.lti.LtiLaunchAuthenticationHandler.cfg`. At the bottom of this file you see the two options `lti.custom_role_name` and `lti.custom_roles`. The former should be set to an LTI role that is supposed to have access to Studio, and the latter is a comma-separated list of Opencast roles that users with this LTI role should get. This list has to at least include `ROLE_STUDIO`.
+
+A minimal configuration giving LTI-`Instructor`-s access to Opencast Studio could look like this:
+
+```
+lti.custom_role_name=Instructor
+lti.custom_roles=ROLE_STUDIO
+```
+
+You will also have to enable persisting LTI users in the same file to avoid access management related failures during ingests initiated by them:
```

Comment:
…during processing ingested videos
```suggestion
You will also have to enable persisting LTI users in the same file to avoid access management related failures during processing ingested video files:
```

Reply:
The ingest will work fine without persisting LTI user reference, because the LTI session exists and the `ROLE_STUDIO` is applied. But to start the workflow, the user will be queried outside of the LTI Session by Opencast. This will fail, if the user (reference) isn't found.

Reply:
The rest is fine for me.

Reply:
This section is simply not true. You don't have to enable persistence. You just have to make sure a user with the stored identifier exists which sounds similar but is something entirely different and Opencast has several different means of archiving that. For instance, you could use the Sakai user provider, or the Moodle user provider, or the Canvas user provider, or LDAP, … More than one of these ways are in active use.

I would drop this whole pull request and even go so far as to drop

> The preferred way to let your users access Studio is via LTI. Remember to configure your LTI users to have the role `ROLE_STUDIO` so that they can access Studio and all APIs used by Studio.

…and replace it with something like…

> Make sure users have the necessary access rights for the ingest service, e.g. by giving them `ROLE_STUDIO` and don't use temporary users since otherwise, workflows may fail due to missing user privileges. When using LTI, you could do this by enabling user persistence in `etc/org.opencastproject.security.lti.LtiLaunchAuthenticationHandler.cfg`.